legal-api/src/legal_api/utils/legislation_datetime.py

```diff
@@ -23,12 +23,12 @@ class LegislationDatetime():
     """Date utility using legislation timezone for reporting and future effective dates."""
 
     @staticmethod
-    def now():
+    def now() -> datetime:
```

Comment:
Hadn't noticed that before.
from legal_api.utils.datetime import datetime
provides the correct **now()**  as **utcnow** so this shouldn't be used.
I can clean this later if needed.

Reply:
Yes, shouldn't be used, but I just cleaned up the type. Please clean up later as needed. Thanks!